src/components/pages/Analytics/TechStackGraph.jsx

```diff
@@ -1,29 +1,26 @@
-import React, { useEffect } from 'react';
+import React, { useEffect, useState } from 'react';
+import axios from 'axios';
 import embed from 'vega-embed';
+//vega-embed allows us to take a graph object, and display it on the ui.
+// for more information visit https://github.com/vega/vega-embed
+
 export default function TechStackGraph() {
+  const [graphData, setGraphData] = useState({ graph: {}, description: '' });
   // let url = `${process.env.REACT_APP_DS_API_URL}/graph/tech-stack-by-role`;
-  let url =
+  const url =
     'http://underdog-devs-ds-a-dev.us-east-1.elasticbeanstalk.com/graph/tech-stack-by-role';
```

Comment:
I think we should not hardcode the url here but instead pull it from our environment variables. It looks like someone had done that earlier in that commented out code on line 9.

Reply:
I have fixed this in [this commit](https://github.com/BloomTech-Labs/underdog-devs-fe-a/pull/325/commits/27335b4e8b355f75fda938537420a710e8d39534)

Thank you for your suggestion!